falcon/testing/helpers.py

```diff
@@ -427,6 +428,10 @@ def closed(self) -> bool:
     def close_code(self) -> int:
         return self._close_code
 
+    @property
+    def close_reason(self) -> str:
+        return self._close_reason
```

Comment:
not sure if we could return str | None and not convert `None` to `''`

Reply:
I followed the way how `close_code()` was written. Should we assume that `close_reason()` will get called before the connection is closed?

Reply:
Ok, reading also close_code it may be fine to have `''` as the default